scripts/lynx/deploy_infraction.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
```

Comment:
Did you want to create one of these for `mainnet` and `tapir`? Or, were you thinking of doing that at the same time as actual `mainnet` deployment (which is fine)?